test/utils/userOp.ts

```diff
@@ -335,7 +362,10 @@ export async function makeEcdsaModuleUserOp(
     },
     userOpSigner,
     entryPoint,
-    "nonce"
+    "getNonce",
```

Comment:
Could you please explain, why we use `getNonce` here while we don't have `getNonce` method in SA?

Reply:
I can leave it as "nonce". It doesn't matter when flag below (useNonceKey) is sent true - it then always uses nonce(uint198) method. passed nonce function name is used if the mentioned flag is false.

Reply:
updated everywhere. all test cases still pass as expected